src/spaceObjects/playerSpaceship.h

```diff
@@ -280,6 +286,55 @@ class PlayerSpaceship : public SpaceShip
     int getWaypointCount() { return waypoints.size(); }
     sf::Vector2f getWaypoint(int index) { if (index > 0 && index <= int(waypoints.size())) return waypoints[index - 1]; return sf::Vector2f(0, 0); }
 
+    // Fleet functions
+    int getFleetCount() { return fleets.size(); }
+    P<SpaceShip> getFleetLeader(int fleet_id) {
+      if (fleet_id > 0 && fleet_id <= int(fleets.size())) {
```

Comment:
I think you have your indentation set up differently from the rest of the project. It should be 4 spaces (and it seems to be just 2)